app/views/components/heading.scala.html

```diff
@@ -14,6 +14,6 @@
  * limitations under the License.
  *@
 
-@(headingKey: String, headingSize: String = "heading-xlarge")(implicit messages: Messages)
+@(headingKey: String, headingSize: String = "heading-xlarge")(messageArgs: Any*)(implicit messages: Messages)
```

Comment:
I needed this trick because the heading for the EORI page needed an argument

Reply:
Instead of passing in `headerKey` and having to do hacks would it not make more sense to pass in the actual value we want as the header?

Reply:
I'd tend to agree with you Ed.

Reply:
e.g we would use it as `heading(messages(key))` rather than `heading(key)`

Reply:
Only arg for keeping it as is is the amount of refactoring of views and tests that would be involved.

Reply:
Looks like @gokyo already refactored them all to add the `()` to the end

Reply:
yes :-) .  He's just finished implementing your suggestion